src/Options.cc

```diff
@@ -106,6 +106,8 @@ void usage(const char* prog, int code)
 	fprintf(stderr, "    -s|--rulefile <rulefile>        | read rules from given file\n");
 	fprintf(stderr, "    -t|--tracefile <tracefile>      | activate execution tracing\n");
 	fprintf(stderr, "    -u|--usage-issues               | find variable usage issues and exit\n");
+	fprintf(stderr, "       --no-usage-warnings          | suppress warnings of unused "
```

Comment:
Yeah, FWIW, I really like that idea.

Reply:
Meaning I should add that with this PR ... or that folks like it as something to add to the to-do list for the future?

Reply:
Given that the option is new in this PR, I'd say yes. On top of that, since 5.0 is about to come out including all of these changes, it'd be good to settle on an argument format now before people are using it instead of changing it between 5.0 and 5.1.

Reply:
I went to do this and then noticed that `-W` is already used (for the "watchdog" - gotta wonder if anyone still actually needs that).  Suggestions?

Reply:
Given that we'd have to deprecate `-W` (which I'm all in favor of!), I propose decoupling these issues, so my latest push does not address that